src/beautifiers/beautifier.coffee

```diff
@@ -97,6 +97,28 @@ module.exports = class Beautifier
       startDir.pop()
     return null
 
+# Retrieves the default line ending based upon the Atom configuration
+  #  `line-ending-selector.defaultLineEnding`. If the Atom configuration
+  #  indicates "OS Default", the `process.platform` is queried, returning
+  #  CRLF for Windows systems and LF for all other systems.
+  # Code modified from atom/line-ending-selector
+  # returns: The correct line-ending character sequence based upon the Atom
+  #  configuration, or `null` if the Atom line ending configuration was not
+  #  recognized.
+  # see: https://github.com/atom/line-ending-selector/blob/master/lib/main.js
+  getDefaultLineEnding: (crlf,lf,optionEol) ->
+    if (!optionEol || optionEol == 'System Default')
```

Comment:
It shouldn't be `OS Default` since `System Default` option will change based on what user set in `Line Ending Selector` package. 

You might call it `Atom Default`, but it could consider as `OS Default` as well if users haven't touched the default setting in that package.